bigchaindb/core.py

```diff
@@ -526,12 +536,15 @@ def write_block(self, block, durability='soft'):
         """
 
         block_serialized = rapidjson.dumps(block)
-        r.table('bigchain').insert(r.json(block_serialized), durability=durability).run(self.conn)
+        self.connection.run(
+                r.table('bigchain')
+                .insert(r.json(block_serialized), durability=durability))
 
     # TODO: Decide if we need this method
```

Comment:
Nevermind this comment.
@rhsimplex is using it to check for duplicated transactions when creating new blocks https://github.com/bigchaindb/bigchaindb/pull/609